src/midi/midi.cpp

```diff
@@ -147,6 +147,8 @@ class MidiState {
 	void Reset()
 	{
 		note_on_tracker.fill(false);
+
+		const uint8_t default_channel_volume = 127;
```

Comment:
Maybe use `max` volume too? (just so people know we're setting the default to the max?)

I remember a previous commit used a max midi volume of 127 -> could maybe dedupe both these uses up to a constant in the header?

Reply:
Yeah, I think that's better, using the max constant for both.